overlays/mingw_w64.nix

```diff
@@ -28,7 +28,19 @@ let
     unset configureFlags
     PORT=$((5000 + $RANDOM % 5000))
     (>&2 echo "---> Starting remote-iserv on port $PORT")
-    WINEDLLOVERRIDES="winemac.drv=d" WINEDEBUG=warn-all,fixme-all,-menubuilder,-mscoree,-ole,-secur32,-winediag WINEPREFIX=$TMP ${wine}/bin/wine64 ${remote-iserv}/bin/remote-iserv.exe tmp $PORT &
+    REMOTE_ISERV=$(mktemp -d)
+    ln -s ${remote-iserv}/bin/* $REMOTE_ISERV
+    for p in $pkgsHostTargetAsString; do
+      find "$p" -iname '*.dll' -exec ln -s {} $REMOTE_ISERV \;
+      find "$p" -iname '*.dll.a' -exec ln -s {} $REMOTE_ISERV \;
+    done
+    (
+    cd $REMOTE_ISERV
+    for l in lib*.dll; do
+      if [[ ! -e "''${l#lib}" ]]; then ln -s "$l" "''${l#lib}"; fi
```

Comment:
Also would be nice to explain lines like this, otherwise they require quite some staring.

Reply:
I have removed the duplication (the same libX.dll -> X.dll mapping was done in windows.nix but really only needs to be done once).  I have added a comment and included `double-conversion` in the test case (so it now fails if this mapping is removed).